lib/libmbedtls/core/sm2-helpers.h

```diff
@@ -0,0 +1,11 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2021 Huawei Technologies Co., Ltd
+ */
+
+#include <mbedtls/bignum.h>
+#include <stddef.h>
+#include <tee_api_types.h>
+
+int sm2_rng_read(void *ignored __unused, unsigned char *buf, size_t blen);
+TEE_Result gen_random_upto(mbedtls_mpi *n, mbedtls_mpi *max);
```

Comment:
I wouldn't mind a `sm2_` prefix for this function too.

Reply:
Sure. Fixed.